doozer/doozerlib/cli/scan_sources_konflux.py

```diff
@@ -267,6 +267,9 @@ async def find_latest_image_builds(self, image_names: typing.List[str]):
             image_names, latest_image_builds)))
 
     async def scan_images(self, image_names: typing.List[str]):
+        # Do not scan images that have been disabled for Konflux operations
+        image_names = filter(lambda name: self.runtime.image_map[name].config.konflux.mode != 'disabled', image_names)
```

Comment:
In addition to this, we have to filter out images with `mode: disabled` as well. Are we already doing that?

Reply:
Yes, `mode: disabled` images are excluded by Doozer. If you try to explicitly request one, you'll get an error like this:
```
The following images or rpms were either missing or filtered out: microshift
```

Reply:
ack thanks